src/test/suite/participant/participant.test.ts

```diff
@@ -1300,21 +1313,6 @@ suite('Participant Controller Test Suite', function () {
             });
           });
 
-          test('without a prompt it asks for the database name without pinging ai', async function () {
```

Comment:
this behavior becomes outdated; unless I am overlooking something, we do not gain too much by asking for the namespace if the user just runs `/schema`. It'd be nicer to show an explainer first to help them discover the syntax.

Reply:
If I remember correctly this was intentional. We want the user to be able to get the schema of a collection by just writing `/schema` and then clicking their namespace.

Reply:
Hm I see. I can see how it's convenient but this does also make it hard for a user to actually discover what the `/schema` command expects as an argument (something which we could definitely instead handle by introducing something like `/help` or equivalent for internal extension info or questions). Seems like the only way to discover that you are meant to ask a question with the `/schema` is through reading documentation.

If we do want to support this way, I could remove the empty message response.